zivid_description/urdf/macros/zivid_camera.xacro

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+    <xacro:macro name="zivid_camera" params="prefix">
+        <!-- Zivid Properties -->
+        <xacro:property name="M_PI" value="3.1415926535897931" />
+
+        <!-- Zivid Base Link -->
+        <link name="${prefix}base_link">
+            <!-- Visuals -->
+            <visual>
+                <origin xyz="-0.0030 -0.0758 0.0445" rpy="${0.5*M_PI} 0 ${0.5*M_PI}"/>
+                <material name="blue">
+                    <color rgba="0.25 0.25 0.25 1"/>
+                </material>
+                <geometry>
+                    <mesh filename="package://zivid_description/meshes/visual/zivid-one-plus.stl" scale="0.001 0.001 0.001"/>
+                </geometry>
+            </visual>
+
+            <!-- Collisions -->
+            <collision>
+                <origin xyz="-0.0030 -0.0758 0.0445" rpy="${0.5*M_PI} 0 ${0.5*M_PI}"/>
+                <geometry>
+                    <mesh filename="package://zivid_description/meshes/collision/zivid-one-plus.stl" scale="0.001 0.001 0.001"/>
```

Comment:
In order to add a collision model, I think it's important that generating the collision model can be done similarly for future cameras, like the Zivid Two. Ideally this should be programatically, but I don't think that is strictly necessary.
Did you generate the collision model in Meshlab through finding the convex hull of the (visual) stl model?

Reply:
Yes, I followed the exact instructions (including the part in the 'Note') from [Create a URDF for an Industrial Robot](http://wiki.ros.org/Industrial/Tutorials/Create%20a%20URDF%20for%20an%20Industrial%20Robot#Collision_quality_STLs) under `1.2.2 Collision quality STLs`. This takes a few minutes to do. I am not sure if this can easily be automated.

Reply:
Ok, I think it's a good way to create the collision model, and reasonably easy to re-produce similarly enough for the Zivid Two as well.
I'll resolve 👍